cityelf/src/main/java/com/cityelf/controller/UserController.java

```diff
@@ -1,20 +1,42 @@
-package com.cityelf.controller;
+package main.java.com.cityelf.controller;
 
-import com.cityelf.model.User;
-import com.cityelf.service.UserService;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.web.bind.annotation.RequestMapping;
-import org.springframework.web.bind.annotation.RestController;
+import main.java.com.cityelf.model.User;
+import main.java.com.cityelf.service.UserService;
+import main.java.com.cityelf.controller.RestController;
+import main.java.org.springframework.beans.factory.annotation.Autowired;
+import main.java.org.springframework.web.bind.annotation.*;
+
+import java.util.List;
 
 @RestController
-@RequestMapping({"/users"})
-public class UserController{
-  @Autowired
-  private UserService us;
-
-  @RequestMapping({"/all"})
-  public java.util.List<User> getAll()
-  {
-    return us.getAll();
-  }
+@RequestMapping("/users")
+public class UserController {
+    @Autowired
+    private UserService userService;
+
+    @RequestMapping("/all")
+    public List<User> getAll() {
+        return userService.getAll();
+    }
+
+    @RequestMapping(value = "/{userId}", method = RequestMethod.GET)
+    public User get(@PathVariable("userId") long id) {
+        return userService.get(id);
+    }
+
+    @RequestMapping(value = "/put", method = RequestMethod.PUT)
+    public void put(@RequestBody User user) {
```

Comment:
Method name would look better as **updateUser** instead of **put**, the same thing for other methods